assets/js/state/sagas/notifications.jsx

```diff
@@ -2,7 +2,13 @@
 import React from 'react';
 import { takeEvery } from 'redux-saga/effects';
 import { toast } from 'react-hot-toast';
-import { NOTIFICATION } from '@state/actions/notifications';
+
+export const NOTIFICATION = 'NOTIFICATION';
```

Comment:
This looks a bit strange. In other components, this would be a slice code. Maybe we should move this piece of code to `state/notifications.js`. Otherwise, in other parts of the code we are importing actions from the saga code, which looks out of place.
What do you think?